samples/ios/Continote/Continote/MyNotesViewController.swift

```diff
@@ -15,39 +15,155 @@
 //
 
 import UIKit
+import Firebase
+import FirebaseDatabaseUI
+import MaterialComponents.MaterialButtons
 
 /**
- This view controller presents the user with a list of their notes in Continote.
- From here the user can open or delete notes.
+ The view controller that presents the user with a list of their Notes in Continote.
+
+ From here the user can add, delete, or open to edit Notes.
  */
 class MyNotesViewController: BaseViewController {
 
+  // Firebase Realtime Database reference for the current user's Notes within Continote.
+  private var notesFirebaseDatabaseRef: DatabaseReference?
+
+  // The data source to populate the TableView of Notes for the current user.
+  private var myNotesTableViewDataSource: MyNotesTableViewDataSource?
+
   // UI outlets
-  @IBOutlet var temporaryTodoLabel: UILabel!
+  @IBOutlet var myNotesTableView: UITableView!
+  @IBOutlet var writeNewNoteButton: MDCRaisedButton!
 
   override func viewDidLoad() {
     super.viewDidLoad()
 
     // Set up this screen's AppBar title.
-    title = Constants.screenTitle
+    title = Constants.Text.screenTitle
+
+    // Style all buttons.
+    writeNewNoteButton.applyAppTheme()
+  }
+
+  override func viewWillDisappear(_ animated: Bool) {
+    super.viewWillDisappear(animated)
+
+    // Remove all bindings and database observers.
+    myNotesTableViewDataSource?.unbind()
+    myNotesTableViewDataSource = nil
+    notesFirebaseDatabaseRef?.removeAllObservers()
+    notesFirebaseDatabaseRef = nil
+  }
+
+  override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+    guard let segueIdentifier = Constants.Segue(rawValue: segue.identifier!) else { return }
```

Comment:
I'd split this up into two:
`guard let identifierString = segue.identifier, let segueIdentifier = Constants.Segue(rawValue: identifierString) else {`

That way you don't have to force unwrap the identifier.